SasquatchTV/SasquatchTV/ViewControllers/CustomPropertyViewController.swift

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import Foundation
+import UIKit
+
+class CustomPropertyViewController : UIViewController, UITableViewDataSource, AppCenterProtocol {
+
+  @IBOutlet weak var keyTextField : UITextField?;
+  @IBOutlet weak var valueTextField : UITextField?;
+  @IBOutlet weak var table : UITableView?;
+
+  var appCenter: AppCenterDelegate!;
+
+  var oldKey : String = "";
+  var oldValue : String = "";
+  var properties : [String : String] = [String : String]();
+
+  override func viewDidLoad() {
+    super.viewDidLoad();
+    table?.dataSource = self;
+    table?.allowsSelection = true;
+    keyTextField?.text = oldKey;
+    valueTextField?.text = oldValue;
+  }
+  
+  @IBAction func addProperty(_ sender: Any) {
+    let propKey : String = String(format : "key%d", properties.count + 1);
+    let propValue : String = String(format : "value%d", properties.count + 1);
+    properties.updateValue(propValue, forKey: propKey);
```

Comment:
should be add instead of update

Reply:
This is an old method implementation, I didn't change it

Reply:
Removed this